imap_processing/ultra/l2/ultra_l2.py

```diff
@@ -541,6 +540,13 @@ def ultra_l2(
     # to "energy" for all instruments.
     map_dataset = map_dataset.rename({"energy_bin_geometric_mean": "energy"})
 
+    # Rename positional uncertainty variables if present
+    if "scatter_theta" in map_dataset and "scatter_phi" in map_dataset:
+        map_dataset = map_dataset.rename(
+            {"scatter_theta": "positional_uncertainty_theta"}
+        )
+        map_dataset = map_dataset.rename({"scatter_phi": "positional_uncertainty_phi"})
```

Comment:
This won't fail if the variables aren't present, so you can just call the rename directly and it will work if they are there.

```suggestion
    # Rename positional uncertainty variables if present
    map_dataset = map_dataset.rename({"scatter_theta": "positional_uncertainty_theta", "scatter_phi": "positional_uncertainty_phi"})
```